_drafts/semantic-release.md

```diff
@@ -0,0 +1,37 @@
+---
+state: In Progress
+---
+
+# Set up release workflow with `semantic-release`
+
+`semantic-release` makes releasing faster, simpler and safer. All of these are valuable if we want to increase public participation in the project and increase momentum and motivation.
+
+## Motivation
+
+Releasing a package is always a bit of a Big Thing™: decisions have to be made about versioning, an error-prone manual release process has to be performed, and somebody has to actually do the work and take the responsibility if something goes wrong. So there are a few barriers to frequent releases, and  many potential sources of error. Automating this process takes away all the uncertainty and hassle out of releasing, makes our version numbers more consistent and reliable, and increases release frequency and project momentum. It also does away with the concept of intentional, manual releases: whenever something lands in master and passes all the tests, `semantic-release` will package an npm release. If we do need to keep control of release timing for marketing reasons, `semantic-release` supports that too.
+
+## Proposed solution
+
+Set up `semantic-release` on `hubotio/hubot` and let it release the npm module for us automatically and continuously.
+
+## Detailed design
+
+An admin/owner [sets up](https://github.com/semantic-release/semantic-release#setup) [semantic-release](https://github.com/semantic-release/semantic-release) on `hubotio/hubot` using `semantic-release-cli setup`. Since we’ve already got tests and Travis set up, this shouldn’t require any additional steps.
+
+We should update `CONTRIBUTING.md`:
+- include more details about our [expected commit message format](https://github.com/semantic-release/semantic-release#default-commit-message-format), and why this is important
```

Comment:
I use **commitizen** with another project that I use semantic-release with. It's a cli wizard (like npm init) to build commit messages in the standard format, which I gather is the convention used by the Angular team and widely adopted since.
https://github.com/commitizen/cz-cli

It's a global module, so I don't know how you'd encourage others to use it, but maybe that's good because you can just make a suggestion for those that need help, otherwise you're still free to write you're own.

The most opaque aspect for me is choosing the "scope". I find it hard to apply consistent logic for my commits.